index.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<html lang="en-us">
+<head>
+    
+    <meta charset="utf-8">
+    
+    <title>Ellie Woods - Bio</title>
+
+      <link href="https://fonts.googleapis.com/css?family=Bungee+Shade|Neuton|Space+Mono:400i,700" rel="stylesheet"> 
+    
+      <link rel="stylesheet" href="CSS/styles.css">
```

Comment:
your styles are not loading because you gave  `href=CSS/styles.css` when your folder for css was `css/styles`. Case matters, in the future try to keep everything lowercase to avoid this issue. 🔨